.ado/pull-request.yml

```diff
@@ -99,8 +99,10 @@ jobs:
       matrix:
         Desktop:
           NuGetId: ReactNative.V8Jsi.Windows
+          PdbExtension: ipdb

```

Comment:
>PdbExtension [](start = 10, length = 12)

[nit] Maybe we can call this "file extensions to ignore" or something like that. The goal was actually to not include PDBs in the UWP build because we're trying to keep its size below the 250Mb limit (which may no longer be needed if we separate the symbols or use the ADO feed instead).

Reply:
You got it. I'll name it `NuGetExcludePattern`.